plugins/multiplexer/ats-multiplexer.cc

```diff
@@ -158,7 +158,7 @@ DoRemap(const Instance &i, TSHttpTxn t)
     generateRequests(i.origins, buffer, location, requests);
     assert(requests.size() == i.origins.size());
 
-    if (is_post_or_put) {
+    if (is_post_or_put || content_length > 0) {
```

Comment:
Note that this is the only production change to support other methods that have Content-Length bodies.